docs/Using-Fleet/MDM-migration-guide.md

```diff
@@ -1,70 +1,67 @@
+4. Migrate your hosts to Fleet in ABM:
+    1. In ABM, unassign the existing hosts' MDM server from the old MDM solution: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Unassign from the current MDM**, and select **Continue**.
+    2. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
+5. In your old MDM solution, unenroll these hosts. MacOS does not allow multiple MDMs to be installed at once. 
 
-2. In ABM, unassign these hosts' MDM server from the old MDM solution: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Unassign from the current MDM**, and select **Continue**.
-
-3. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
+Once the above steps are completed, the end user now needs to complete a few steps. The **My Device** page in Fleet Desktop will present end users with instructions to turn on MDM. 
 
-4. In your old MDM solution, unenroll these hosts. MacOS does not allow multiple MDMs to be installed at once. This step is required to present end users with instructions to turn on MDM in Fleet.
-
-5. The **My Device** page in Fleet Desktop will present end users with instructions to turn on MDM. Share [these guided instructions](#instructions-for-end-users) with your end users.
+Fleet has created [these guided instructions](#instructions-for-end-users) that can be shared with your end users.
 
 ## FileVault recovery keys
 
 _Available in Fleet Premium_
 
 In Fleet, you can enforce FileVault (disk encryption) to be on. If turned on, hosts’ disk encryption keys will be stored in Fleet. Learn how [here](./MDM-macOS-settings.md#disk-encryption).
 
-During migration from your old MDM solution, disk encryption will be turned off for your macOS hosts until they are enrolled to Fleet and MDM is turned on for these hosts.
+If your hosts did not have disk encryption turned on under the old MDM, there is no migration action needed. When you turn on disk encryption, the host will be encrypted and the key will be escrowed to Fleet automatically.
+
+If the host had disk encryption turned on under the old MDM, disk encryption will be turned off for your macOS hosts until they are enrolled to Fleet and MDM is turned on for these hosts. Your end users will need to take an action to reset their disk encryption key for Fleet to be able to store the key. 
```

Comment:
I don't fully understand this line. To me, it reads as if: If I have filevault turned on under the old MDM, then my Mac will become unencrypted when I migrate to Fleet. It will only be encrypted again once I turn FileVault back on via Fleet. 

However, my understanding is that the Mac hard drive would continue to be encrypted. But Fleet will not have access to the key. Fleet can only catch a key if you reset filevault. I think this is different from what is described here. Which is it?